src/allotropy/parsers/unchained_labs_lunatic/unchained_labs_lunatic_structure.py

```diff
@@ -63,40 +64,64 @@ def _create_measurement(
         )
 
     measurement_identifier = random_uuid_str()
+    error_documents: list[ErrorDocument] = []
     calculated_data.extend(
-        _get_calculated_data(well_plate_data, wavelength_column, measurement_identifier)
+        _get_calculated_data(
+            well_plate_data, wavelength_column, measurement_identifier, error_documents
+        )
     )
+    absorbance = well_plate_data.get(float, wavelength_column)
+
+    if absorbance is None:
+        error_documents.append(
+            ErrorDocument(
+                error=NOT_APPLICABLE,
+                error_feature=DETECTION_TYPE.lower(),
+            )
+        )
     return Measurement(
         type_=MeasurementType.ULTRAVIOLET_ABSORBANCE,
         device_type=DEVICE_TYPE,
         detection_type=DETECTION_TYPE,
         identifier=measurement_identifier,
         detector_wavelength_setting=float(wavelength),
         electronic_absorbance_reference_wavelength_setting=background_wavelength,
-        electronic_absorbance_reference_absorbance=background_absorbance,
-        absorbance=well_plate_data.get(float, wavelength_column, NaN),
+        measurement_custom_info={
+            "electronic_absorbance_reference_absorbance": background_absorbance
```

Comment:
```suggestion
            "electronic absorbance reference absorbance": background_absorbance
```

Also - it appears that `measurement_custom_info` is not being used in the schema mapper - not sure why it was added to the dataclass but not used - but you'll want to add the code to use it so this doesn't get dropped

Reply:
It is being used in the `_get_luminescence_measurement_document`. Other technique measurement documents could use when needed.

Reply:
See comment below about value that is being removed